libs/state/util-queries/src/lib/use-react-query-ws-subscription/use-react-query-ws-subscription.ts

```diff
@@ -76,7 +76,7 @@ export function useReactQueryWsSubscription({
         websocket.close()
       }
     }
-  }, [queryClient, getAccessTokenSilently, onMessage, url, searchParams])
+  }, [queryClient, getAccessTokenSilently, onMessage, url, searchParams.toString()])
```

Comment:
toString() really neccesary ?

Reply:
Yes it is because we don't want to compare `URLSearchParams` object. We want to compare the result string